bson/bsonoptions/time_codec_options.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package bsonoptions
```

Comment:
nit: Can this package be called `options` instead of `bsonoptions`? It's a subpackage of `bson` so `bsonoptions` seems repetitive.

Reply:
There's already a package in mongo called options, so it would be annoying for users if they need both and they're both named options.